README.md

```diff
@@ -2,6 +2,8 @@
 
 dokku-docker-auto-volumes is a plugin for [dokku][dokku] that automatically persist docker volumes declared in the Dockerfile.
 
+/var/lib/dokku/volumes 
```

Comment:
Well the goal of this plugin is to not bother with declaring anything :) But it works only with Dockerfiles that declares properly their volumes (with the `VOLUME` command). Then when deploying such an app, after the initial build of the Dockerfile, the plugin will create directories for each volume declared in the docker image and copy the files (if any) from the docker image to those directories. After that it will use `dokku docker-options` to automatically add the `-v` option for each volume (for `run` and `deploy` phases).